src/index.d.ts

```diff
@@ -20,26 +20,26 @@ declare module 'react-native-onesignal' {
         hasPrompted             ?: boolean;   // ios
```

Comment:
This file should not even be here since it should be generated automatically. I think this commit is unnecessary

Reply:
Done, removed this commit